packages/web-pkg/src/components/FilesList/ResourceTiles.vue

```diff
@@ -82,7 +82,7 @@
               class="oc-flex-inline oc-p-s"
               :disabled="isResourceDisabled(resource)"
               :model-value="isResourceSelected(resource)"
-              @click.stop.prevent="toggleTile([resource, $event])"
+              @click.stop.prevent="toggleTile([resource, $event], $event)"
```

Comment:
`$event` is already being passed to `toggleTile()`, no need to pass it twice, no?